azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# Copy pasted from:
+# https://github.com/MicrosoftDocs/pipelines-android/blob/master/azure-pipelines.yml
+
+# Android
+# Build your Android project with Gradle.
+# Add steps that test, sign, and distribute the APK, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/android
+
+pool:
+    vmImage: 'macos-latest'
+
+steps:
+    - task: Gradle@2
+      inputs:
+          workingDirectory: ''
+          gradleWrapperFile: 'gradlew'
+          gradleOptions: '-Xmx3072m'
+          javaHomeOption: 'JDKVersion'
+          jdkVersionOption: '1.8'
+          jdkArchitectureOption: 'x64'
+          publishJUnitResults: true
+          testResultsFiles: '**/TEST-*.xml'
+          tasks: 'assembleDebug test'
+      displayName: gradlew assembleDebug test
+
+    - task: CopyFiles@2
+      inputs:
+          contents: '**/*.apk'
```

Comment:
consider updating this to bundle in the future

Reply:
good catch.
I am using bundle.
It is a copy paste error LOL!!!!!

Reply:
We are already generating apk as well.
I will leave this task for the next step on this epic.
This task is only to run and configure azure pipelines as fast as I can.

I will close this conversation.

Reply:
the horse way!